### PR TITLE
[11.0] [ADD] Add account_tag_menu module.

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

File: account_tag_menu/models/account_tag.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2017 FOREST AND BIOMASS ROMANIA SA
```
Comment (elicoidal) [inline account_tag_menu/models/account_tag.py]: any reason the date is different between the files?

**Review by elicoidal — APPROVED** [review]
LGTM
(UT+Demo data would be nice-to-have)

**Comment by feketemihai** [comment]
I saw it but waited for comments to push again, update when i arrive at the office.

**Comment by feketemihai** [comment]
@elicoidal Thanks for the review, i updated and added demo data. Thanks again.

**Review by elicoidal — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Review by AaronHForgeFlow — COMMENTED** [review]
:+1:

**Pull request merged**